frontend/appflowy_flutter/lib/plugins/database_view/grid/presentation/widgets/row/action.dart

```diff
@@ -16,16 +16,22 @@ import '../../layout/sizes.dart';
 class RowActions extends StatelessWidget {
   final String viewId;
   final RowId rowId;
+  final String? groupId;
   const RowActions({
     required this.viewId,
     required this.rowId,
+    this.groupId,
```

Comment:
Wondering if this should be required. If it isn't required, does that mean the card can't be duplicated?

Reply:
`RowCard` is also used in calendar which does not have groups.

Reply:
Makes sense to me. It might be worth it to keep the `RowActions` API this way and implement two factory constructors for `RowCard` to prevent API misuse.

Reply:
Got it